File: Selection_Sort/selection_sort.js
const selectionSort = (arr) => {
    let minIdx;
    for(let i = 0; i < arr.length - 1; i++) {
        minIdx = i;
        for(let j = i + 1; j < arr.length ; j++) {
            if(arr[minIdx] > arr[j]) {
                minIdx = j;
            }
        }
        let temp = arr[i];
        arr[i] = arr[minIdx];
        arr[minIdx] = temp;  
    }
}

// Driver code to test above
let lst = [6, 1, 5, 9, 9, 5, 3, 8, 0]
selectionSort(lst)
console.log(lst)

File: Selection_Sort/SelectionSort.py
# Implementation of Selection Sort
def selection_sort(arr: list) -> None:
    for i in range(len(arr)):
        min_idx = i
        for j in range(i + 1, len(arr)):
            if arr[min_idx] > arr[j]:
                min_idx = j
        arr[i], arr[min_idx] = arr[min_idx], arr[i]
            

# Driver code to test above
if __name__ == "__main__":
    nums = [6, 1, 5, 9, 9, 5, 3, 8, 0]
    selection_sort(nums)
    print(nums)
